pkgs/development/compilers/vlang/default.nix

```diff
@@ -1,20 +1,20 @@
 { lib, stdenv, fetchFromGitHub, glfw, freetype, openssl, makeWrapper, upx, boehmgc, xorg, binaryen, darwin }:
 
 let
-  version = "weekly.2023.44";
+  version = "weekly.2023.48";
```

Comment:
On a previous upgrade someone suggested rebasing to version releases only. Version releases seem to come out every 1-2 months. What do you think?

Reply:
There are so many fixes which are pushed in weekly releases. The next release may take 3-4 months from now on.

Reply:
> There are so many fixes which are pushed in weekly releases. The next release may take 3-4 months from now on.

It's more like 1-2 months. 0.4.3 was 11 Nov, 0.4.2 30th Sep, 0.4.1 3rd Sep, 0.4 1st July. So just under one a month.
But if you want to push upgrades more frequently than that then by all means stick to weekly releases.

Reply:
IMO there should be two versions of vlang in nixpkgs then.
https://github.com/vlang/v/releases/tag/0.4.3 should be regular ``vlang``.
https://github.com/vlang/v/releases/tag/weekly.2023.48 should be ``vlang-weekly``.
You could go even father and make ``vlang-git`` that just pulls master directly as well.

Reply:
> IMO there should be two versions of vlang in nixpkgs then. https://github.com/vlang/v/releases/tag/0.4.3 should be regular `vlang`. https://github.com/vlang/v/releases/tag/weekly.2023.48 should be `vlang-weekly` (`vlang-beta` might work as well). You could go even farther and make `vlang-git` that just pulls master directly as well.
> 
> Regardless, this PR should be using `0.4.3`, since base derivations are supposed to be using stable releases.

That's even better, yeah.

> > Regardless, this PR should be using `0.4.3`, since base derivations are supposed to be using stable releases.
> 
> I have a closed PR with 0.4.3 which I can re-open. Or this one can be rebased to 0.4.3.

I will close this PR instead.